packages/common/src/store/pages/audio-rewards/selectors.ts

```diff
@@ -36,6 +36,14 @@ export const getUndisbursedUserChallenges = (state: CommonState) =>
     ).includes(challenge.specifier)
   })
 
+export const getUndisbursedUserChallengeBySpecifier = createSelector(
```

Comment:
nit: A comment or example of how this is used might be helpful. I don't know that I understand it just by the function name :-)